features/__init__.pyi

```diff
@@ -219,3 +219,6 @@ REFERRERS_API: FeatureNameValue
 
 SUPERUSERS_FULL_ACCESS: FeatureNameValue
 ASSIGN_OAUTH_TOKEN: FeatureNameValue
+
+# Feature Flag: If set to true, supports setting notifications on image expiry
+FEATURE_IMAGE_EXPIRY_TRIGGER: FeatureNameValue
```

Comment:
you'll need to remove the `FEATURE_` prefix

Reply:
good catch, fixed it